backend/backend/settings/base.py

```diff
@@ -155,7 +155,7 @@
 STATIC_ROOT = os.path.join(BASE_DIR, "staticfiles")
 
 # This section is added to support serving the frontend files
-#INSTALLED_APPS.extend(["whitenoise.runserver_nostatic"])
+INSTALLED_APPS.extend(["whitenoise.runserver_nostatic"])
```

Comment:
This either belongs directly in the definition of `INSTALLED_APPS` above (line 48), or, if it's only needed for e.g. production, in the respective settings file for that environment.

Reply:
I agree. To be honest I don't know if it is needed at all. It was in the original MS code and I think I commented it out at some point. If it works I'll revert this change and we can figure out if whitenoise is even necessary in the future.